server.js

```diff
@@ -0,0 +1,88 @@
+const net = require("net");
+const files = require("./files.js");
+
+var today = new Date();
```

Comment:
You want to move this date into the createServer callback function. The way it currently is, the date will never change. So if your server was up for a few days, it'll always return the date when the server was started and not the current time.